src/js/common/schemaform/FormApp.jsx

```diff
@@ -231,11 +72,19 @@ class FormApp extends React.Component {
   }
 }
 
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict got checked in.

Reply:
Yep! Just pushed the fix for that